controllers/TransitionBase.js

```diff
@@ -39,14 +39,14 @@ define(["dcl/dcl", "dojo/when", "dojo/Deferred", "dojo/promise/all", "../Control
 					viewData: event.viewData,
 					reverse: event.reverse,
 					transition: event.transition,
-					displayDeferred: event.displayDeferred,
+					displayResolve: event.displayResolve,
 					doingPopState: event.doingPopState,
 					viewParams: event.viewParams,
 					dapp: {}
 				});
 			},
 
-			_loadViewsInOrder: function (viewPaths, i, event, syncDef) {
+			_loadViewsInOrder: function (viewPaths, i, event, syncResolve) {
```

Comment:
I converted it to use a promise in wkeese/dapp@f46a5bd15e0a8599d93967560c1046b3804524c3.

Probably you can pull it via

git pull https://github.com/wkeese/dapp.git lie2